generate/names.go

```diff
@@ -137,7 +137,7 @@ func joinPrefixList(prefix *prefixList) string {
 // comment), but it's used to construct both the type-names from the input and
 // the next prefix-list.
 func typeNameParts(prefix *prefixList, typeName string) *prefixList {
-	// GraphQL types are conventionally UpperCamelCase, but it's not required;
+	// GraphQL's types are conventionally UpperCamelCase, but it's not required;
```

Comment:
I think this is clearer (and correct) as-is?  I think of them as "GraphQL types"; "GraphQL" is the kind of type they are, not the owner of the type.